staging/src/k8s.io/apiserver/pkg/admission/plugin/resourcequota/resource_access_test.go

```diff
@@ -121,5 +125,124 @@ func TestLRUCacheLookup(t *testing.T) {
 			}
 		})
 	}
+}
+
+// TestGetQuotas ensures we do not have multiple LIST calls to the apiserver
+// in-flight at any one time. This is to ensure the issue described in #22422 do
+// not happen again.
+func TestGetQuotas(t *testing.T) {
+	var (
+		testNamespace1              = "test-a"
+		testNamespace2              = "test-b"
+		listCallCountTestNamespace1 int64
+		listCallCountTestNamespace2 int64
+	)
+	resourceQuota := &corev1.ResourceQuota{
+		ObjectMeta: metav1.ObjectMeta{
+			Name: "foo",
+		},
+	}
+
+	resourceQuotas := []*corev1.ResourceQuota{resourceQuota}
+
+	liveLookupCache := lru.New(100)
```

Comment:
we do not to create the variable explicitly now, right?

Reply:
true, nice catch.